docs/spec.md

```diff
@@ -132,6 +139,66 @@ Informs the client that channels are no longer available.
 }
 ```
 
+### Parameter Values
+
+This message is sent as a response to the client's [Get Parameters request](#get-parameters). Only supported if the server declares the `parameters` [capability](#server-info).
+
+#### Fields
+
+- `op`: string `"parameterValues"`
+- `parameters`: array of:
+  - `name`: string, name of the parameter
+  - `value`: number | boolean | string | number[] | boolean[] | string[]
+  - `meta`: object, only given if requested with `includeMetaInfo: true`
+    - `type`: bool | int64 | float64 | string | byte[] | bool[] | int64[] | float64[] | string[]
```

Comment:
Do you think all systems that might want to implement parameter support will have this same kind of meta information? Why bother making it optional? Is the type info redundant with the `value` since you can just check the JSON type? (I guess the specific type may matter for numbers, but Im' not sure what the use case is)

nit: the possible values for `type` should all be wrapped in quotes, since they are strings, right? or just "`type`: `string`"?

Why bother nesting type under `meta`? Are there other things we will want to include in `meta`?

Reply:
> Do you think all systems that might want to implement parameter support will have this same kind of meta information?

The meta field is probably very system specific. I intended it as a struct where the server could communicate additional parameter information such as the type, description, min / max / default values, etc. I'm not sure if we should enforce a schema on that or if we keep it schema-less. In the end, the `meta` info is optional, clients will still be able to display parameter values without interpreting it